components/elm/src/biogeochem/FireMod.F90

```diff
@@ -305,11 +305,12 @@ subroutine FireArea (bounds, &
            if (pi <=  col_pp%npfts(c)) then
               p = col_pp%pfti(c) + pi - 1
               ! For crop veg types
-              if( veg_pp%itype(p) > nc4_grass )then
+              if( crop(veg_pp%itype(p)) == 1 .or. iscft(veg_pp%itype(p)) == 1 )then
```

Comment:
Another location where == is used instead of >= for iscft. Also see lines 335 and 511 in this subroutine.

Reply:
I would like to go through and do the conversion to logical types for a couple of other variables, including crop(), in the same way the I have done for iscft(). I would like to hold off and bring that in with other code cleanup in another PR.